### PR TITLE
fix(components): fix cursor state on `EmptySelectorButton`

### PR DESCRIPTION
**Pull request by ncdiehl11**
# Overview

When hovering an enabled `EmptySelectorButton`, (ex: walking through creating a protocol in PD redesign and adding pipettes/modules), the cursor should be `pointer`.

Closes RQA-3197

## Test Plan and Hands on Testing

- turn on PD redesign
- create a new protocol 
- walk through steps and verify that at steps 2 and 4, hovering the active `EmptySelectorButton` for "Add a pipette and tips" and individual module additions produces a pointer cursor

## Changelog

add cursor style to `EmptySelectorButton`

## Review requests

see test plan

## Risk assessment

low

### CODE REVIEW

File: components/src/atoms/buttons/EmptySelectorButton.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ export function EmptySelectorButton(
     border: none;
     width: ${buttonSizing};
     height: ${buttonSizing};
+    cursor: ${disabled ? 'default' : 'pointer'};
```
Comment (koji) [inline components/src/atoms/buttons/EmptySelectorButton.tsx]: if you merge edge or rebase, you will be able to use the followings.

```suggestion
    cursor: ${disabled ? CURSOR_DEFAULT : CURSOR_POINTER};
```

Comment (koji) [reply components/src/atoms/buttons/EmptySelectorButton.tsx]: https://github.com/Opentrons/opentrons/pull/16136

**Review by jerader — APPROVED** [review]
lgtm after addressing Koji's comment

File: components/src/atoms/buttons/EmptySelectorButton.tsx (hunk 1 above)
Comment (ncdiehl11) [reply components/src/atoms/buttons/EmptySelectorButton.tsx]: thanks Koji

**Pull request merged**